lua_cjson.c

```diff
@@ -75,6 +75,13 @@
 #define DEFAULT_DECODE_INVALID_NUMBERS 0
 #endif
 
+// fix when compile under Microsoft Visual Studio, the compiler will complain that unable to find the reference of strncasecmp
```

Comment:
@momofarm Just see the existing comments in the current C source file. We use the `/* ... */` form consistently. The `// ...` form was just added in recent C standards.